test/dao.suite/patch-by-id.suite.js

```diff
@@ -0,0 +1,188 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: loopback-datasource-juggler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+module.exports = function(dataSource, should, connectorCapabilities) {
+  describe('patchById', function() {
+    var personId, Person, StubUser;
+    before(function setupDatabase(done) {
+      Person = dataSource.define('Person', {
+        name: String,
```

Comment:
Please make `name` required and add a test for `patchById(id, { age: 15 })`. See one of the earlier comments.